index.js

```diff
@@ -515,6 +526,11 @@ function getSidebarProps() {
 * Shows the next sidebar for the current track i.e. values or utility
 */
 function toggleSidebar() {
+    // anytime the user opens a sidebar, remove the previous 3 week destroy timer
+    timers.clearInterval(destroyTimer);
+    // start a new 3 week destroy timer
+    startDestroyTimer(1814400000);
```

Comment:
We should probably define this as a variable like the others so we can make it configurable via `config.json`?